docs/UserGuide.md

```diff
@@ -128,6 +128,22 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Locating persons by tag : `findtag`
+
+Find persons with the given tag.
+
+Format: `findtag KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g tag will match Tag
+* The order of the keywords does not matter. e.g. Tag One will match One Tag
+* Only the tag is searched
+* Only full words will be matched e.g. Ta will not match Tag
```

Comment:
I think it's better to standardise whether to put the keyword passed and a tag in backticks, which was done below. I think it looks better with backticks. :grin: